packages/pygments/meta.yaml

```
@@ -0,0 +1,9 @@
+package:
+  name: pygments
```

```suggestion
  name: Pygments
```

This needs to match the tarball name.  Also, the `packages/pygments` directory needs to move to `packages/Pygments`.

---

Updated. Thanks!